test/controllers/WebSocketClient.java

```diff
@@ -50,24 +45,24 @@ public Throwable getThrowable() {
         }
 
         public void onOpen(WebSocket websocket) {
-            //logger.info("onClose: ");
-            //websocket.sendMessage("hello");
+            // do nothing
         }
 
-        public void onClose(WebSocket websocket) {
-            //logger.info("onClose: ");
+        @Override
+        public void onClose(WebSocket webSocket, int i, String s) {
+            // do nothing
         }
 
         public void onError(Throwable t) {
-            //logger.error("onError: ", t);
+            // do nothing
             throwableFound = t;
         }
 
         @Override
-        public void onMessage(String s) {
+        public void onTextFrame(String payload, boolean finalFragment, int rsv) {
             //logger.info("onMessage: s = " + s);
-            onMessageCallback.accept(s);
+            onMessageCallback.accept(payload);
```

Comment:
There's a couple of changes in this file caused by a rather large [WebSocket revamp](https://github.com/AsyncHttpClient/async-http-client/commit/98bef40637212fc1c64ec47b1b11bb1a15144704) between versions of AHC.